format/view-spec.md

```diff
@@ -107,7 +107,7 @@ This type of representation stores the original view definition in SQL and its S
 | Required/Optional | Field Name | Description |
 |-------------------|------------|-------------|
 | Required | type | A string indicating the type of representation. It is set to "sql" for this type. |
-| Required | sql | A string representing the original view definition in SQL |
+| Required | query | A string representing the original view definition in SQL |
```

Comment:
why not change the API to `sql()`? I think it feels more consistent there since the API class is `SQLViewRepresentation`

Reply:
Yeah good point, in my mind query was more broader than just SQL but yeah ultimately this is for SQLViewRepresentation which will by definition just be SQL. I'll close this and raise a PR to update the API